meson.build

```diff
@@ -0,0 +1,45 @@
+project('protobuf-c', 'c', 'cpp', version : '1.3.0', license : 'protobuf-c')
```

Comment:
The license should be "Apache-2" or "BSD" or whatever one (or ones) they are using.